subsys/bluetooth/shell/gatt.c

```diff
@@ -159,10 +159,6 @@ int cmd_gatt_discover(int argc, char *argv[])
 	discover_params.end_handle = 0xffff;
 
 	if (argc < 2) {
```

Comment:
Remove the goto done; completely while you are at it?
if ( argc > 1 ) { /* UUID handling */ }

Reply:
Done